src/main/java/frc/robot/DriveTrain.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.wpilibj.drive.MecanumDrive;
+
+public class DriveTrain {
+    
+    private final CANSparkMax driveFrontLeft = new CANSparkMax(4, MotorType.kBrushless);
+    private final CANSparkMax driveRearLeft = new CANSparkMax(3, MotorType.kBrushless);
+    
+    private final CANSparkMax driveFrontRight = new CANSparkMax(2, MotorType.kBrushless);
+    private final CANSparkMax driveRearRight = new CANSparkMax(1, MotorType.kBrushless);
+
+    private final MecanumDrive robotDrive = new MecanumDrive(driveFrontLeft, driveRearLeft, driveFrontRight, driveRearRight);
+
+    public DriveTrain() {
+        //Left Invert
+        driveFrontLeft.setInverted(false);
+        driveRearLeft.setInverted(false);
+        //Right Invert
+        driveFrontRight.setInverted(true);
+        driveRearRight.setInverted(true);
+    }
+
+    public void mecanumDrive(double xSpeed, double ySpeed, double zRotation) {
+        robotDrive.driveCartesian(-ySpeed, xSpeed, zRotation);
```

Comment:
I think negating the y value would be more appropriate to do when you’re reading the inputs. Then all the input modifications can happen in the same place.